src/SourceBuild/content/repo-projects/package-source-build.proj

```diff
@@ -60,7 +91,7 @@
 
     <!-- Content of the .version file to include in the tarball -->
     <ItemGroup>
-      <VersionFileContent Include="$(RepositoryCommit);$(MicrosoftSourceBuildIntermediateInstallerVersion)" />
+      <VersionFileContent Include="$(RepositoryCommit);$(SdkVersion)" />
```

Comment:
I think the name of the artifacts tarball file should use the same version.

Reply:
I did discuss this with @MichaelSimons few days ago - the desire was keep the current non-stable versioning of private source-build artifacts.

Reply:
What's the reasoning for that?

Reply:
My thinking is that all the SB artifacts like the artifacts, prebuilts, smoke-test-prereqs, are a type of non-shipping artifact that should have a version number unique per the build they came from.  The stable versions will be the same throughout all of the servicing builds.  Thinking about this further, perhaps the .version file should contain both?

Reply:
Going back to the original issue that prompted this file to be added - https://github.com/dotnet/source-build/issues/3124.  In the use case, the desire is to link the artifacts to the build that produced it.  A stable version would not be useful for this purpose.  This raises a question on what the use case is for the stable version versus the build version?

Reply:
Is the current behavior correct and the test should be udated?

Reply:
Would the `.version` file be extended to now include unique version, like this?
```
<sha>
<stable SDK version>
<unique build version>
```

Reply:
> Going back to the original issue that prompted this file to be added - [dotnet/source-build#3124](https://github.com/dotnet/source-build/issues/3124). In the use case, the desire is to link the artifacts to the build that produced it. A stable version would not be useful for this purpose. This raises a question on what the use case is for the stable version versus the build version?

Then why wouldn't this also apply to the name of the SDK tarball file? I think they should be consistent at least.

Reply:
To me this comes down to whether or not the artifact is a shipping/end user artifact.  The SDK tarball is an end user/shipping artifact with the afore mentioned artifacts are not.

Reply:
How can it not be considered a shipping artifact when the VMR directly references it for PSB artifacts?

Reply:
It is a bit of a grey area to me.  The MSFT build uses the MSFT version but distro maintainers don't use the MSFT one.  They each bring their own.  It is infra like arcade.  Arcade isn't considered a shipping artifact yet it is required for building.

Reply:
Ok, I'm on board with that. In that case, we should keep everything the same as it was originally but update the test to not make on an exact version because the SDK's .version file will be stable and not match the version from the artifacts exactly. Instead the test could just match on the `<major>.<minor>.<patch>` portion of the version.

Reply:
> Ok, I'm on board with that. In that case, we should keep everything the same as it was originally but update the test to not make on an exact version because the SDK's .version file will be stable and not match the version from the artifacts exactly. Instead the test could just match on the `<major>.<minor>.<patch>` portion of the version.

I'll create another PR for that change and we can merge whichever one has consensus.

Reply:
I think we can do even better.

`.version` file in SDK has the following contents:
```
<sha>
<stable version>
<rid>
<real unique version>
```

i.e. in my local build this was:
```
fed94d5f58d11fe5c4eb6ba07904dd3775b21e6c
8.0.101
fedora.38-x64
8.0.101-servicing.23559.1
```

`.version` file in PSB artifacts tarball has the following content:
```
<sha>
<real unique version>
```

In my local build this was:
```
0000000000000000000000000000000000000000
8.0.101-servicing.23559.1
```

So, we could just modify this line to use 3rd index instead of 1st: https://github.com/dotnet/installer/blob/97fc62d5800005e4c54778afca8c41f041a6ad0e/src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/SourceBuiltArtifactsTests.cs#L59

Reply:
New PR: https://github.com/dotnet/installer/pull/17819